DROD/Main.cpp

```diff
@@ -1470,14 +1467,10 @@ bool IsAppAlreadyRunning()
 	tmp += wszSlash;
 	tmp += wszDROD;
 	tmp += dotpid;
-	const UINT lflen = tmp.length();
-	if (!(lockfile = new char[lflen + 1])) return true;
```

Comment:
@TomBrouws , note that if there is any expansion in the number of characters due to the UCS-16 to UTF8 conversion, the char buffer size here would prove insufficient and could result in a crash.